Make matlab_pkg fails on MacOS

I've checkout out the current `master` and tried to run `make matlab_pkg` on MacOS 10.14.6 (Mojave). Got the error:

```shell
...
Converting texinfo to Matlab-style documentation: doctest.m
cp -ra inst/private/*.m tmp/doctest-matlab-0.7.0+/private/
cp: the -R and -r options may not be specified together.
make: *** [tmp/doctest-matlab-0.7.0+] Error 1
```

`-a` on MacOS equals `-pPR` according to `man cp`. A workaround is to install `coreutils` e.g. via Homebrew and then run `env PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH" make matlab_pkg`

In some way related to #216.

--- a/Makefile
+++ b/Makefile
@@ -129,12 +129,12 @@ matlab_pkg: $(MATLAB_PKG_ZIP)
 ${MATLAB_PKG}: | $(BUILD_DIR) ${MATLAB_PKG}/private
 	$(OCTAVE) --path ${CURDIR}/util --silent --eval \
 		"convert_comments('inst/', '', '../${MATLAB_PKG}/')"
-	cp -ra inst/private/*.m ${MATLAB_PKG}/private/
-	cp -ra COPYING ${MATLAB_PKG}/
-	cp -ra CONTRIBUTORS ${MATLAB_PKG}/
-	cp -ra NEWS ${MATLAB_PKG}/
-	cp -ra README.matlab.md ${MATLAB_PKG}/
-	cp -ra test ${MATLAB_PKG}/
+	cp -a inst/private/*.m ${MATLAB_PKG}/private/
+	cp -a COPYING ${MATLAB_PKG}/
+	cp -a CONTRIBUTORS ${MATLAB_PKG}/
+	cp -a NEWS ${MATLAB_PKG}/
+	cp -a README.matlab.md ${MATLAB_PKG}/
+	cp -a test ${MATLAB_PKG}/
 
 matlab_test:
 	cd "${MATLAB_PKG}"; ${MATLAB} -nojvm -nodisplay -nosplash -r "${TEST_CODE}"

